tests/functions/test_origin_request.py

```diff
@@ -609,83 +649,46 @@ def test_origin_request_cookie_uri(
     ]
     assert request["headers"]["set-cookie"] == [
         {
-            "value": "CloudFront-Key-Pair-Id=K1MOU91G3N7WPY; Secure; "
-            "HttpOnly; SameSite=lax; Domain=ex.cloudfront.net; "
-            "Path=/content/; Max-Age=43200"
+            "value": "CloudFront-Key-Pair-Id=XXXXXXXXXXXXXX; Secure; "
+            "HttpOnly; SameSite=lax; Domain=http://localhost:8049; "
+            "Path=/content/*; Max-Age=1800"
```

Comment:
This is indicating a mistake, the Path element of the cookie should still be just `/content/` and not `/content/*` as there is nothing about wildcards in the standard for cookies.

I think I pointed this out in the exodus-gw code review already so I guess it's just that you'd copied the query string from an earlier version of that code, but it'd still be good to fix this.

Reply:
Hmm Okay. It is present in exodus-gw as well but can change it there before it's deployed.